src/date-range-picker/time-offset.ts

```diff
@@ -39,11 +39,12 @@ export function shiftTimeOffset(
 
   /*
     This regex matches an ISO date-time with
+    - optionally matches date-only format
     - optional seconds;
     - optional milliseconds;
     - optional time offset or 'Z'.
   */
-  const dateTimeRegex = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}(:\d{2})?(\.\d{1,3})?(((\+|-)\d{2}(:\d{2})?)|Z)?$/;
+  const dateTimeRegex = /^\d{4}-\d{2}-\d{2}(T\d{2}:\d{2}(:\d{2})?(\.\d{1,3})?(((\+|-)\d{2}(:\d{2})?)|Z)?)?$/;
```

Comment:
we do we even need the shiftTimeOffset for date-only values?

Reply:
Removed this changes as it is no longer needed after this change https://github.com/cloudscape-design/components/pull/3111/files?w=1